content/en/internal/getting-started/_index.md

```diff
@@ -1,14 +1,25 @@
 ---
 title: "Getting Started"
 linkTitle: "Getting Started"
+description: "Introduction to internal EGI services"
 type: docs
 weight: 10
 ---
 
-Please find the list of internal services on the
-[EGI website](https://www.egi.eu/internal-services/).
+Use this section to get started quickly with internal EGI services:
+
+- The [complete list](https://www.egi.eu/internal-services) of internal
+  EGI services shows what supports the coordination of the EGI federation
+- The [**Configuration Database**](https://goc.egi.eu) records the topology
+  of the sites in the EGI federation
+- [**Service Monitoring**](http://argo.egi.eu) tracks and controls the
+  performance of the services
+- [**Accounting**](https://accounting.egi.eu) tracks service and resource
+  usage, providing insights and reports on consumption
+- The [**Helpdesk**](http://helpdesk.egi.eu) lets users and providers report
+  incidents and bugs, or request changes
 
 ## Request for information
 
-You can ask for more information about the services at:
-[here](https://www.egi.eu/more-information/).
+You can look [here](https://www.egi.eu/more-information) for
+more information about the EGI services.
```

Comment:
I tend to find ask for more information more clear and relevant as the page is just a form allowing to ask for more information.

Reply:
OK